core/src/main/scala/pureconfig/DerivedReaders.scala

```diff
@@ -75,14 +75,15 @@ trait DerivedReaders1 {
     hint: ProductHint[Wrapped]): WrappedDefaultValue[Wrapped, FieldType[K, V] :: T, Option[V] :: U] = new WrappedDefaultValue[Wrapped, FieldType[K, V] :: T, Option[V] :: U] {
 
     override def fromConfigObject(co: ConfigObject, default: Option[V] :: U): Either[ConfigReaderFailures, FieldType[K, V] :: T] = {
-      val keyStr = hint.configKey(key.value.toString().tail)
+      val fieldName = key.value.toString().tail
```

Comment:
I know that this is not new code, but this is not the best way to convert a `Symbol` to a `String`. We are calling its `toString` and dropping the single quote, while we could access its string value directly by doing `key.value.name`.